setup.py

```diff
@@ -15,6 +15,11 @@
 for entry in dependency_links:
     os.system("pip install {}".format(entry))
 
-setup(name='humblerl',
+setup(name='HumbleRL',
       version='0.0.1',
+      description='Straightforward reinforcement learning Python framework',
+      author='Piotr Januszewski',
```

Comment:
To nie powinno się składać z nas trzech już w tym momemncie? Zacząłem ja, ale rozwijamy to razem. Są jakieś inne znaczniki, żeby wskazać kto rozwija? Jak nie to trzeba nas trzech wpisać po przecinku.